R-package/src/lightgbm_R.h

```diff
@@ -441,14 +440,18 @@ LIGHTGBM_C_EXPORT SEXP LGBM_BoosterGetPredict_R(
 
 /*!
 * \brief make prediction for file
-* \param handle handle
+* \param handle Booster handle
 * \param data_filename filename of data file
 * \param data_has_header data file has header or not
-* \param is_rawscore
-* \param is_leafidx
+* \param is_rawscore 1 to get raw predictions, before transformations like
+*                    converting to probabilities, 0 otherwise
+* \param is_leafidx 1 to get record of which leaf in each tree
+*                   observations fell into, 0 otherwise
+* \param is_predcontrib 1 to get feature contributions, 0 otherwise
+* \param parameter additional parameters
```

Comment:
Missed `start_iteration`. And `num_iteration` should be after `parameter`.

Reply:
> And num_iteration should be after parameter.

I don't think that's correct. `num_iteration` comes before `parameter` in this function.

https://github.com/microsoft/LightGBM/blob/11b77471cf02c17e1cd43cc8f4b2ddf71be4c2b2/R-package/src/lightgbm_R.h#L460-L462

Reply:
> I don't think that's correct.

My mistake, sorry! I don't know why I wrote this while was thinking about the opposite.